Documentation/PackedAssetBuilds.md

```diff
@@ -1,12 +1,40 @@
 # Packaged MultiplayerSample Builds
+
 To make relocatable client and server builds for the MultiplayerSample, we recommend making packaged builds. These package builds will contain the Game or Server Launcher and the bundled assets needed to run the launcher outside of the developer environment.
 
 You can make both release packaged builds or profile packaged builds. For more information about creating release builds, see the O3DE documentation on [Creating a Project Game Release Layout for Windows](https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/).
 
 The guide below covers how to make profile packaged builds which are very useful for early sharing and play testing.
 
+## Brief outline of the packaging steps
+
+1. Install WWise SDK (required for o3de-multiplayersample but optional for other projects)
+1. Use the engine from GitHub, not the installer
+1. Create a build for non-monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile non-monolithic mode
+1. Process all the assets using the Asset Processor from profile non-monolithic mode
+1. Create game bundles (.pak files) from the seed lists (a list of game assets)
+1. Create a build for monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile (and/or release) monolithic mode
+1. Create a folder for the packaged build outside of the engine, this will be the share-able folder of the packaged game
+1. Copy the monolithic game binaries and the game bundles to the packaged folder
+1. The packaged build is ready and can be shared without requiring the developer environment.
+
+## Pre-requisites
+
+*Important*: you can not use an installer to produce packaged builds. In order to produce a packaged build you will need to build the engine from source from GitHub.
+
+Additionally, we will need to compile the engine in non-monolithic profile build and in monolithic profile (and optionally release) mode.
+
+> Note, a monolithic build is a special build of O3DE engine and projects that combines all the gems used by the project into a single game executable. Additionally, monolithic builds do not build the Editor or the Asset Processor. Thus, a regular non-monolithic build is used to process assets and bundle them into .pak files, while a monolithic build is used to produce share-able game binaries.
+
+This guide uses `C:\git\o3de` is the source for the O3DE, cloned from GitHub.
+`C:\git\o3de-multiplayersample` is the location of the clone project. These paths are optional. If you choose to use different paths, amend the command scripts in the rest of the guide accordingly.
```

Comment:
```suggestion

1. Your computer is [setup to build O3DE](https://www.o3de.org/docs/user-guide/build/)
1. You have the O3DE engine source code from GitHub [https://github.com/o3de/o3de](https://github.com/o3de/o3de)
 
*Important*: you can not use an O3DE SDK installer to produce packaged builds, that is why you need the engine source code from GitHub.

NOTE: This guide uses `C:\git\o3de` as the location where the O3DE engine source code has been cloned from GitHub.
`C:\git\o3de-multiplayersample` is the location of where the project has been cloned from GitHub. These paths are optional. If you use different paths, amend the commands in the rest of the guide accordingly.
```

Reply:
The reason I put it this way is because I want to call this requirement out early in the document and then re-iterate later on.